package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stylelint-config-wolox",
-  "version": "1.0.10",
+  "version": "1.0.11",
```

Comment:
I'd bump the minor instead of the patch number. I wouldn't classify this as a "fix", or at least it impacts the projects more than a bugfix